src/did/src/init.rs

```diff
@@ -47,4 +47,12 @@ impl Default for EvmCanisterInitData {
 }
 
 /// These are the arguments which are taken by the signature verification canister init fn
-pub type SignatureVerificationCanisterInitData = Vec<Principal>;
+#[derive(Debug, Clone, Serialize, CandidType, Deserialize)]
+pub struct SignatureVerificationCanisterInitData {
+    /// Access list of principals that are allowed to send transactions to the EVM canisters
+    pub access_list: Vec<Principal>,
+    /// EVM canisters that receive transactions from the signature verification canister
+    pub evm_canisters: Vec<Principal>,
```

Comment:
What happens if more than one evm is provided? will it send the same transactions to all of them?

Reply:
ok,thanks